tests/Unit/Helpers/PointwiseFunctions/PostNewtonian/Python/Bindings.cpp

```diff
@@ -0,0 +1,53 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include <array>
+#include <pybind11/numpy.h>
+#include <pybind11/pybind11.h>
+#include <pybind11/stl.h>
+#include <string>
+
+#include "Helpers/PointwiseFunctions/PostNewtonian/BinaryTrajectories.hpp"
+#include "Utilities/ErrorHandling/SegfaultHandler.hpp"
+
+namespace py = pybind11;
+
+PYBIND11_MODULE(_Pybindings, m) {  // NOLINT
+  enable_segfault_handler();
+  py::class_<BinaryTrajectories>(m, "BinaryTrajectories")
+      .def(py::init<double, const std::array<double, 3>&, bool>(),
+           py::arg("initial_separation"),
+           py::arg("center_of_mass_velocity") =
+               std::array<double, 3>{{0.0, 0.0, 0.0}},
+           py::arg("newtonian") = false)
+      .def("separation", py::vectorize(&BinaryTrajectories::separation),
+           py::arg("time"))
+      .def("orbital_frequency",
+           py::vectorize(&BinaryTrajectories::orbital_frequency),
+           py::arg("time"))
+      .def("angular_velocity",
+           py::vectorize(&BinaryTrajectories::angular_velocity),
+           py::arg("time"))
+      .def("positions", &BinaryTrajectories::positions, py::arg("time"))
+      .def(
+          "positions",
+          [](const BinaryTrajectories& binary_trajectories,
+             const py::array_t<double> time) -> py::array_t<double> {
```

Comment:
If you're already going to be doing this in C++, why not just add it to the BinaryTrajectories class.

Reply:
Added commits for this